src/tools/mmaps_generator/PathGenerator.cpp

```diff
@@ -84,11 +100,23 @@ bool handleArgs(int argc, char** argv,
                 return false;
 
             float maxangle = atof(param);
-            if (maxangle <= 90.f && maxangle >= 45.f)
+            if (maxangle <= 90.f && maxangle >= 0.f)
                 maxAngle = maxangle;
             else
                 printf("invalid option for '--maxAngle', using default\n");
         }
+        else if (strcmp(argv[i], "--maxAngleNotSteep") == 0)
+        {
+            param = argv[++i];
+            if (!param)
+                return false;
+
+            float maxangle = atof(param);
```

Comment:
You can use this
https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/common/Utilities/StringConvert.h

Reply:
These changes are made after completing the maps.